test/e2e-common.sh

```diff
@@ -27,6 +27,8 @@ source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 
 function install_operator_resources() {
   echo ">> Deploying Tekton Operator Resources"
+
+  E2E_TEST_PLATFORM=${E2E_TEST_PLATFORM:-kubernetes} \
   make apply || fail_test "Tekton Operator installation failed"
```

Comment:
`make TARGET=E2E_TEST_PLATFORM apply` and you are good to go, no need for changes in the `Makefile`